src/pages/UserProfile.js

```diff
@@ -0,0 +1,397 @@
+import {useState, useEffect, useCallback} from 'react';
+import {find, isNil, isNumber} from 'lodash';
+import {button, div, form, h, h2, hr, input, label, p, a} from 'react-hyperscript-helpers';
+import {eRACommons} from '../components/eRACommons';
+import {PageHeading} from '../components/PageHeading';
+import {Notification} from '../components/Notification';
+import {FormField, FormFieldTypes, FormValidators} from '../components/forms/forms';
+import {Institution, Support, User} from '../libs/ajax';
+import {NotificationService} from '../libs/notificationService';
+import {Storage} from '../libs/storage';
+import { Notifications, getPropertyValuesFromUser, isEmailAddress} from '../libs/utils';
+
+export default function ResearcherProfile(props) {
+  const [isLoading, setIsLoading] = useState(true);
+  const [profile, setProfile] = useState({
+    profileName: '',
+    institutionId: undefined,
+    suggestedInstitution: undefined,
+    selectedSigningOfficialId: undefined,
+    suggestedSigningOfficial: undefined,
+    eraCommonsId: undefined,
+    email: undefined,
+    id: undefined
+  });
+
+  const possibleSupportRequests = [
```

Comment:
I think this is a candidate for pruning since it is only for a component that isn't on this page any more.